components/bitly/actions/create-bitlink/create-bitlink.mjs

```diff
@@ -0,0 +1,64 @@
+import common from "../../common/common.mjs";
+
+export default {
+  key: "bitly-create-bitlink",
+  name: "Create a Bitlink",
+  description:
+    "Converts a long url to a Bitlink and sets additional parameters.",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    bitly: {
+      type: "app",
+      app: "bitly",
+    },
+    long_url: {
+      type: "string",
+      description: "URL to shorten",
+    },
+    title: {
+      type: "string",
+      optional: true,
+      description: "Bitlink title",
+    },
+    tags: {
+      type: "string[]",
+      optional: true,
+    },
+    domain: {
+      type: "string",
+      optional: true,
+      description:
+        "Bitlinks that contain deeplinks configured with a custom domain",
```

Comment:
Can you modify this description slightly to tell the user what to put as the domain here? As a user reading this, I'm not sure I understand what value to put for the domain

Reply:
is this better? Bitlinks that contain deeplinks configured with a custom domain. e.g. bit.ly